source/MaterialXGraphEditor/RenderView.cpp

```diff
@@ -452,7 +452,7 @@ void RenderView::updateMaterials(mx::TypedElementPtr typedElem)
         if (typedElem)
         {
             mx::NodePtr node = typedElem->asA<mx::Node>();
-            materialNode = node && node->getType() == mx::MATERIAL_TYPE_STRING ? node : nullptr;
+            materialNode = node;
```

Comment:
Could we take this a step further and remove the `materialNode` alias for `node`?